src/overlays/actors/ovl_En_St/z_en_st.c

```diff
@@ -563,7 +563,9 @@ s32 EnSt_DecrStunTimer(EnSt* this) {
     if (this->stunTimer == 0) {
         return 0;
     }
-    this->stunTimer--; //! @bug  no return but v0 ends up being stunTimer before decrement
+    this->stunTimer--;
+    //! @bug No return, v0 ends up being stunTimer before decrement.
+    //! The return value is not used so it doesn't matter.
```

Comment:
I feel like we can have this comment without assuming prior knowledge of asm stuff like knowing v0 is for returns

Something like "No value is explicitly returned. The value that ends up being returned (based on asm register placement) is `this->stunTimer`"

Reply:
I'm not totally sure I agree and we already have comments similar to this elsewhere, I think comments like this one can be reworked later if desired

Reply:
I mean not everyone using the codebase will have this prior knowledge that v0 is for returns. Ofc everyone involved with actual decomp does 😄 
But ok, we can revist later I guess